src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -151,6 +154,51 @@ export default App.extend({
 
     this.showList();
   },
+  subscribe() {
+    const channel = Radio.channel('ws');
+    const filterKey = this.getState().getType();
+
+    channel.request('subscribe', this.collection.models, { filters: { [filterKey]: this.filters } });
+
+    this.listenTo(channel, 'message', ({ category, resource, payload }) => {
+      const modelResource = category === 'ResourceCreated' ? payload.resource : resource;
+      const model = Radio.request('entities', 'get:store', modelResource);
+
+      if (this.collection.get(model) || model.isFetching()) return;
+
+      if (filterKey === 'flows') {
+        this._fetchFlow(model);
+        return;
+      }
+
+      if (filterKey === 'actions') {
```

Comment:
I think you'll need to remove the `if` here and have a default, otherwise we'll have an uncovered `else` branch

Reply:
Updated 👍